cirq-superstaq/cirq_superstaq/daily_integration_test.py

```diff
@@ -239,6 +239,21 @@ def test_supercheq(service: css.Service) -> None:
     assert fidelities.shape == (32, 32)
 
 
+def test_dfe(service: css.Service) -> None:
+    circuit = cirq.Circuit(cirq.H(cirq.q(0)))
+    target = "ss_unconstrained_simulator"
+    ids = service.submit_dfe(
+        rho_1=(circuit, target),
+        rho_2=(circuit, target),
+        m=5,
```

Comment:
A bit hard to know what this is based on arg name. How about
```suggestion
        num_of random_bases=5,
```
?

Reply:
Done.